.github/workflows/rubyonrails.yml

```diff
@@ -55,6 +55,61 @@ jobs:
       - name: Run tests
         run: bin/rails test
 
+  system_tests:
+    runs-on: ubuntu-latest
+    container:
+      image: ruby:3.2.2
+    services:
+      postgres:
+        image: postgres:16.1
+        ports:
+          - "5432:5432"
+        env:
+          POSTGRES_DB: hostedgpt_test
+          POSTGRES_USER: rails
+          POSTGRES_PASSWORD: password
+    env:
+      RAILS_ENV: test
+      DATABASE_URL: "postgres://rails:password@postgres:5432/hostedgpt_test"
+    steps:
+      - run: |
+          set -x
+          apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4EB27DB2A3B88B8B
+          echo "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google.list >/dev/null
+          apt-get update
+          apt-get install -y google-chrome-stable jq curl unzip
```

Comment:
I'm kind of surprised there isn't a docker image with this already installed, but I don't have any concerns with gettin git working this way too

Reply:
There's a github action for it, but it's broken. That's where I pulled these lines from.

Reply:
I was actually able to resolve this -- the github action for chromedriver wasn't working because we were running our tests in a container, and the action wasn't compatible with that.